CHANGES/8062.feature

```diff
@@ -0,0 +1 @@
+Added server capability to check for static files with Brotli compression via a ``.br`` extension -- by :user:`steverep`
```

Comment:
Let's use a full stop at the end of the sentence.
```suggestion
Added server capability to check for static files with Brotli compression via a ``.br`` extension -- by :user:`steverep`.
```
I recently updated the change note writing style guidelines to make them more clear: https://docs.aiohttp.org/en/stable/contributing.html#adding-change-notes-with-your-prs.